en/power_module/can_pmu.md

```diff
@@ -0,0 +1,79 @@
+# CAN PMU
```

Comment:
Can you put CUAV in name here and in summary. Also can you please name the FILE to **cuav_can_pmu.md** so that if another company has a similar name for their product it is easy to differentiate.